src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,249 @@
+# a: the factor on the gaussian curve that determines the height of the curve
+# x_part: the list of x-locations in mv-ns which were passed to each gaussian fit to determine the curve.
+def gauss_estimate(import_data_results, peak_results, gauss_width=10):
+  #import data
+  amax=import_data_results["amax"]
+  amin=import_data_results["amin"]
+  bin_width=import_data_results["bin_width"]
+  area=import_data_results["area"]
+  nbins=import_data_results["nbins"]
+  n=import_data_results["n"]
+  h=import_data_results["h"]
+  peak_bin_numbers=peak_results["peak_bin_numbers"]
+  peaks=peak_results["peaks"]
+
+  gauss_dict={"peak":[],"mu":[],"sigma":[],"integral":[],"a":[],"x_part":[]}
+  x = np.linspace(amin+bin_width/2, amax-bin_width/2, nbins)
+  
+  #loop through each peak to attempt to fit gaussian to each
+  for idpeak, peak_bin_number in enumerate(peak_bin_numbers):
+    #set limits for what data gets passed to each gaussian fitting for each peak
+    gauss_width=gauss_width
+    min_bin=peak_bin_number-gauss_width
+    if(min_bin<0):
+      min_bin=0
+    max_bin=peak_bin_number+gauss_width+1
+    if(max_bin>(nbins-1)):
+      max_bin=nbins-1
+    x_part=x[min_bin:max_bin]
+    n_part=n[min_bin:max_bin]
+
+    #fit gaussian curve
+    popt, _ = curve_fit(gauss, x_part, n_part,p0=[10000,peaks[idpeak],50])
+ 
+    #results from gaussian fit
+    a=popt[0]
+    x0=popt[1]
+    sigma=popt[2]
+
+    # Only save data from good gaussian fits
+    if abs(x0-peaks[idpeak])<=5*bin_width and sigma<=10*bin_width:
+      gauss_dict["integral"].append(integrate.quad(lambda z:gauss(z,a,x0,sigma),x_part[0],x_part[-1])[0])
+      gauss_dict["peak"].append(peaks[idpeak])
+      gauss_dict["mu"].append(x0)
+      gauss_dict["sigma"].append(sigma)
+      gauss_dict["a"].append(a)
+      gauss_dict["x_part"].append(x_part)
+    
+  return gauss_dict
+
+#- this function plots the histogram data plus overlays all of the accepted gaussian curves
+#- import_data_results is the dictionary of results returned by import_data
+#- gauss_dict is the dictionary of results returned by gauss_estimate
+def plot_gauss_estimate(import_data_results,gauss_dict):
+  #import data
+  h=import_data_results["h"]
+  
+  #plot regular data
+  fig,ax = plot.make_simple_figure() 
+  plot.plot_data1d(ax=ax, data=h, label='Data', histtype='fill')  
+  
+  #plot gaussians
+  for idpeak, peak in enumerate(gauss_dict["peak"]):
+    plt.plot(gauss_dict["x_part"][idpeak], gauss(gauss_dict["x_part"][idpeak],gauss_dict["a"][idpeak],gauss_dict["mu"][idpeak],gauss_dict["sigma"][idpeak]), label='Fit of Peak at '+str(int(peak)))
+  
+  ##TODO: for some reason running this messes with the plot, it ends up making the whole plot really thin. My issue is that the curve_fit requires using numpy arrays not ax like the other plotting, I'm not sure how to make them more compatible. 
+  #plot.add_std_label(ax=ax, label='Preliminary', rlabel='Lowlight Data with Fitted Gaussian Curves for Parameter Estimation')
```

Comment:
Notice that the `rlabel` will be attempted to be inserted text in the top right corner of the plot, a very long string will inevitably change how "top right" is defined, as the string display dimensions is longer than the figure dimensions, which is what is causing this to break. 

In general, avoid long labels on plot, as the full description should either be in text form in the display material (like a paper draft or notebook descripts), or the description should be broken down into smaller components to be placed in the legends.